evalml/model_understanding/force_plots.py

```diff
@@ -0,0 +1,100 @@
+                                              cls_dict["expected_value"],
+                                              matplotlib=matplotlib)
+    return shap_plots
+
+
+def force_plot(pipeline, rows_to_explain, training_data, y):
+    """ Function to generate the data required to build a force plot.
+
+    Args:
+        pipeline (PipelineBase): The pipeline to generate the force plot for.
+        rows_to_explain (list(int)): A list of the indices of the training_data to explain.
+        training_data (pandas.DataFrame): The data used to train the pipeline.
+        y (pandas.Series): The target data.
+
+    Returns:
+        list(dict()): list of dictionaries where each dict contains force plot data.  Each dictionary
+            entry represents the explanations for a single row.
+            e.x. For single row binary force plots:
+                    [{'malignant': {'expected_value': 0.37,
+                                    'feature_names': ['worst concave points', 'worst perimeter', 'worst radius'],
+                                    'shap_values': [0.09, 0.09, 0.08],
+                                    'plot': AdditiveForceVisualizer}]
+                 For two row binary force plots:
+                    [{'malignant': {'expected_value': 0.37,
+                                    'feature_names': ['worst concave points', 'worst perimeter', 'worst radius'],
+                                    'shap_values': [0.09, 0.09, 0.08],
+                                    'plot': AdditiveForceVisualizer},
+                    {'malignant': {'expected_value': 0.29,
+                                    'feature_names': ['worst concave points', 'worst perimeter', 'worst radius'],
+                                    'shap_values': [0.05, 0.03, 0.02],
+                                    'plot': AdditiveForceVisualizer}]
+
+    Raises:
+        TypeError: if rows_to_explain is not a list.
+        TypeError: if all values in rows_to_explain aren't integers.
+    """
+
+    if not isinstance(rows_to_explain, list):
+        raise TypeError("rows_to_explain should be provided as a list of row index integers!")
+    if not all([isinstance(x, int) for x in rows_to_explain]):
+        raise TypeError("rows_to_explain should only contain integers!")
+
+    explanations = []
+    prediction_explanations = explain_predictions(pipeline, training_data, y, rows_to_explain,
+                                                  top_k_features=len(training_data.columns), include_shap_values=True,
+                                                  output_format="dict")
+    row_explanations = prediction_explanations["explanations"]
+    for row_explanation in row_explanations:
+        row_exp = row_explanation["explanations"]
+        row_exp_dict = {}
+        for cls_exp in row_exp:
+            cls = cls_exp["class_name"] if cls_exp["class_name"] is not None else "regression"
```

Comment:
Simplification: use the [dict `get` method](https://docs.python.org/3/library/stdtypes.html#dict.get)
```python
cls = cls_exp.get("class_name", "regression")
```

Reply:
Haha you got me on this one!  "class_name" always exists, but it's the *value* of the "class_name" key that is None.  The .get() function is for if the "class_name" key doesn't exist.  You taught me something here :)